packages/calling-component-bindings/src/errorBarSelector.ts

```diff
@@ -114,10 +115,24 @@ export const errorBarSelector: ErrorBarSelector = createSelector(
       }
     }
 
-    if (diagnostics?.media.latest.cameraPermissionDenied?.value === true) {
+    /* @conditional-compile-remove(unsupported-browser) */
+    /**
+     * show the Mac specific strings if the platform is detected as mac
+     */
+    if (
+      diagnostics?.media.latest.cameraPermissionDenied?.value === true &&
+      environmentInfo?.environment.platform === 'mac'
+    ) {
       activeErrorMessages.push({ type: 'callMacOsCameraAccessDenied' });
     }
-    if (diagnostics?.media.latest.screenshareRecordingDisabled?.value === true) {
+
+    /**
+     * This UFD only works on mac still so we should only see it fire on mac.
+     */
+    if (
+      diagnostics?.media.latest.screenshareRecordingDisabled?.value === true &&
+      /* @conditional-compile-remove(unsupported-browser) */ environmentInfo?.environment.platform === 'mac'
+    ) {
```

Comment:
Is it ok that this fix will only be stable once the unsupportedBrowser feature moves to stable?

Reply:
yup, since currently we cannot block screen sharing on windows for now that this will never fire. however that might not always be the case so we should still proceed to add this fix since the string is specific to mac.

Reply:
This fires when you start a screenshare but then cancel on the "choose what to share" dialog on windows -- but it exists in stable releases currently so don't consider this a blocking comment for this PR

Reply:
More info: https://teams.microsoft.com/l/message/19:5e150d250f724d1089c949305efdc8cc@thread.skype/1674868535410?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e9c1fc3-39df-4486-a26a-456d80e80f82&parentMessageId=1674687915063&teamName=Azure%20Communication%20Services&channelName=Web%20UI%20-%20Team&createdTime=1674868535410